src/Core__Date.res

```diff
@@ -1,14 +1,12 @@
 type t = Js.Date.t
 
-module Time = {
-  type t = float
-}
+type time = float
```

Comment:
This is a confusing name for what this actually is. It isn't any more or less a "time" than `t`, just a different representation. The closest proper name for the format I think is "Unix time".

Reply:
To be explicit, maybe it could be called `millisSinceEpoch`.

Reply:
Yeah, I think that would be better. Since it's no longer a module I don't think it matters much that it's a bit wordy, although ideally I think it also ought to be a private type so that it's distinct from a `float`, which suggests it should be a module so that it can also have a conventional constructor function. But communicating the concept clearly is more important.

Reply:
Can also add more information about what it is exactly in a docstring.